packages/editor/src/lib/editor/Editor.ts

```diff
@@ -8552,7 +8564,6 @@ export class Editor extends EventEmitter<TLEventMap> {
 						// If the user is in pen mode, but the pointer is not a pen, stop here.
 						if (!isPen && isPenMode) return
 
-						// If we've started panning, then clear any long press timeout
```

Comment:
We used to have `clearTimeout(this._longPressTimeout)` here in the past, lookse like a stale comment.